kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/KafkaConnectService.java

```diff
@@ -328,6 +332,6 @@ private Mono<KafkaConnectClientApi> withConnectClient(KafkaCluster cluster, Stri
             .filter(connect -> connect.getName().equals(connectName))
             .findFirst())
         .switchIfEmpty(Mono.error(ConnectNotFoundException::new))
-        .map(KafkaConnectClients::withKafkaConnectConfig);
+        .map(config -> KafkaConnectClients.withKafkaConnectConfig(config, maxBuffSize));
```

Comment:
I think we should refactor this code in general: lets
1. make KafkaConnectClients normal object (spring component), CACHE field will become non static. (I would personally rename class to KafkaConnectClientsFactory)
2. inject maxBuffSize value to it, not to KafkaConnectService